pom.xml

```diff
@@ -173,7 +173,7 @@
 		<uimaj.version>3.4.1</uimaj.version>
 		<morfologik.version>2.1.9</morfologik.version>
 		<checkstyle.plugin.version>3.2.0</checkstyle.plugin.version>
-		<onnxruntime.version>1.16.3</onnxruntime.version>
+		<onnxruntime.version>1.17.0</onnxruntime.version>
```

Comment:
@jzonthemtn Why is it 1.17.0 here, different from the title of the issue?

Reply:
Looks like a typo because https://onnxruntime.ai/docs/reference/releases-servicing.html says 1.17.0 is the last one.

Reply:
@mawiesne Good eye! `1.17.1` is what I meant here. The page @rzo1 had does say `1.17.0` but there is a `1.17.1` artifact.